components/x-gift-article/src/HighlightRecipientMessage.scss

```diff
@@ -0,0 +1,66 @@
+@import '@financial-times/o-colors/main';
+@import '@financial-times/o-spacing/main';
+
+.highlight-recipient-message {
+  .highlight-message-content {
+    position: relative;;
+    background-color: oColorsByName('slate');
+    color: oColorsMix('ft-grey', 'white', 20);
+    padding: oSpacingByName('s3');
+  
+    .title {
+      @include oTypographySans($scale: 1, $weight: 'regular');
+      margin: 0;
+      color: oColorsByName('white');
+    }
+  
+    .description {
+      @include oTypographySans($scale: 0);
+      margin: 0;
+    }
+    
+    .divider{
+      padding-top: oSpacingByName('s2');
+      padding-bottom: oSpacingByName('s3');
+      &::after {
+        border-top: oSpacingByName('s1') solid oColorsByName('mint');
+        content: "";
+        position: absolute;
+        width: 40px;
+        content: "";
+        display: block;
+      }
+    }
+  
+    .close {
```

Comment:
We need to follow the [BEM naming convention](https://css-tricks.com/bem-101/) to avoid clashes with other CSS in product teams pages. For example this would be something like `.x-gift-article__highlight-recipient-message__close`